src/frontend/komponenter/Fagsak/Vedtak/OppsummeringVedtak.tsx

```diff
@@ -252,42 +258,10 @@ const OppsummeringVedtak: React.FunctionComponent<IVedtakProps> = ({ åpenBehand
                         </Button>
                     </div>
 
-                    <Modal
-                        open={visModal}
-                        onClose={() => settVisModal(false)}
-                        closeButton={true}
-                        shouldCloseOnOverlayClick={false}
-                    >
-                        <Modal.Content>
-                            <Heading size={'medium'} level={'2'}>
-                                Totrinnskontroll
-                            </Heading>
-                            <Modaltekst>Behandlingen er nå sendt til totrinnskontroll</Modaltekst>
-                            <Knapperad>
-                                <Button
-                                    key={'oppgavebenk'}
-                                    variant={'secondary'}
-                                    size={'medium'}
-                                    onClick={() => {
-                                        settVisModal(false);
-                                        navigate('/oppgaver');
-                                    }}
-                                    children={'Gå til oppgavebenken'}
-                                />
-                                <KnappHøyre
-                                    key={'saksoversikt'}
-                                    variant={'secondary'}
-                                    size={'medium'}
-                                    onClick={() => {
-                                        settVisModal(false);
-                                        navigate(`/fagsak/${fagsakId}/saksoversikt`);
-                                        window.location.reload();
-                                    }}
-                                    children={'Gå til saksoversikten'}
-                                />
-                            </Knapperad>
-                        </Modal.Content>
-                    </Modal>
+                    <BehandlingSendtTilTotrinnskontrollModal
+                        visModal={visModal}
+                        settVisModal={settVisModal}
+                    />
```

Comment:
Tenker denne gjør seg bedre i egen fil

Reply:
🥇

Reply:
Tenker at du kan gjøre samme endring med modalen som ligger i `LeggTilEllerFjernBrevmottakere`. Trekk modalen ut i egen fil som får `visModal` og `settVisModal` inn som props ⭐ da kan du gjenbruke den her og i filen den ligger i nå

Reply:
Vet ikke om vi henter så mye på det. LeggTilEllerFjernBrevmottakere har jo nesten bare Modalen i seg. I tillegg har modalen skjema under seg, så vi må fremdeles sende med en del parametere

https://github.com/navikt/familie-ba-sak-frontend/pull/2548/files

Reply:
Tenkte på at det må til for å få på plass "Se detaljer"-knappen som er med i skissene

Reply:
Ah 🤦‍♂️Den gikk meg hus forbi! Har lagt det inn nå